src/Cameras/Inputs/BaseCameraPointersInput.ts

```diff
@@ -0,0 +1,291 @@
+                    previousPinchSquaredDistance = pinchSquaredDistance;
+                }
+            }
+        };
+
+        this._observer = this.camera.getScene().onPointerObservable.add(
+            this._pointerInput,
+            PointerEventTypes.POINTERDOWN | PointerEventTypes.POINTERUP |
+            PointerEventTypes.POINTERMOVE);
+
+        this._onLostFocus = () => {
+            pointA = pointB = null;
+        };
+
+        element.addEventListener("contextmenu",
+            <EventListener>this.onContextMenu.bind(this), false);
+
+        Tools.RegisterTopRootEvents([
+            { name: "blur", handler: this._onLostFocus }
+        ]);
+    }
+
+    /**
+     * Detach the current controls from the specified dom element.
+     * @param element Defines the element to stop listening the inputs from
+     */
+    public detachControl(element: Nullable<HTMLElement>): void {
+        if (this._onLostFocus) {
+            Tools.UnregisterTopRootEvents([
+                { name: "blur", handler: this._onLostFocus }
+            ]);
+        }
+
+        if (element && this._observer) {
+            this.camera.getScene().onPointerObservable.remove(this._observer);
+            this._observer = null;
+
+            if (this.onContextMenu) {
+                element.removeEventListener("contextmenu", <EventListener>this.onContextMenu);
+            }
+
+            this._onLostFocus = null;
+        }
+    }
+
+    /**
+     * Gets the class name of the current input.
+     * @returns the class name
+     */
+    public getClassName(): string {
+        return this._className;
+    }
+
+    /**
+     * Get the friendly name associated with the input class.
+     * @returns the input friendly name
+     */
+    public getSimpleName(): string {
+        return "pointers";
+    }
+
+    /**
+     * Called on pointer POINTERDOUBLETAP event.
+     * Override this method to provide functionality on POINTERDOUBLETAP event.
+     */
+    protected doDoubleTouch(type: string) {
+    }
+
+    /**
+     * Called on pointer POINTERMOVE event if only a single touch is active.
+     * Override this method to provide functionality.
+     */
+    protected doTouch(pointA: Nullable<PointerTouch>,
+                      offsetX: number,
+                      offsetY: number): void {
+    }
+
+    /**
+     * Called on pointer POINTERMOVE event if multiple touches are active.
+     * Override this method to provide functionality.
+     */
+    protected doMultiTouch(pointA: Nullable<PointerTouch>,
+                           pointB: Nullable<PointerTouch>,
+                           previousPinchSquaredDistance: number,
+                           pinchSquaredDistance: number,
+                           previousMultiTouchPanPosition: Nullable<PointerTouch>,
+                           multiTouchPanPosition: Nullable<PointerTouch>): void {
+        /* if (previousPinchSquaredDistance === 0 && previousMultiTouchPanPosition === null) {
```

Comment:
Just meant as documentation on how to use this method.
I tried writing it in the docstring but code was much simpler.

How do you want this?
option 1: Un-comment the `if` statement but leave the text inside the block.
option 2: Remove the whole comment; `FollowCameraPointersInput` already gives an example.

Reply:
Was thinking about this some more;
I don't like the idea of 1. (un-commenting) as it would take up space even when minified.

That leaves 2. (Remove the whole comment).

Agree?

Reply:
agree on 2